com.unity.formats.alembic/Editor/Misc/EditorHelper.cs

```diff
@@ -0,0 +1,14 @@
+namespace UnityEditor.Formats.Alembic.Importer
+{
+    internal class EditorHelper
+    {
+        internal static string BuildPathIfNecessary(string relativePath)
```

Comment:
Not worth it, I've done this only to stay retro-compatible with 19.3 and 20.3, which are no longer supported. Otherwise, I would have embedded this into the caller scope, that is being tested by almost the entire test suite.